rubygems.md

```diff
@@ -32,21 +32,7 @@ Some points to note:
 
 ## Releasing gem versions
 
-This should be done using the [`gem_publisher`
-gem](https://github.com/alphagov/gem_publisher).  See `gem_publisher`'s
-[README](https://github.com/alphagov/gem_publisher#readme) for details on
-setting this up.
-
-## Jenkins build script
-
-This should delete the `Gemfile.lock` before each build to ensure that the gem
-is being tested against the versions specified in the gemspec.
-
-An example `jenkins.sh`:
-
-    #!/bin/sh
-    set -e
-    rm -f Gemfile.lock
-    bundle install --path "${HOME}/bundles/${JOB_NAME}"
-    bundle exec rake
-    bundle exec rake publish_gem
+The default Jenkins build script will automatically detect the presence of a
```

Comment:
It's not clear to me what "default" is adding here. It might be useful to refer to the actual script, e.g. "The buildProject function in the shared govuk_jenkinslib.groovy file will automatically...".